dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/EPersonRestRepository.java

```diff
@@ -182,42 +182,22 @@ public EPersonRest findByEmail(@Parameter(value = "email", required = true) Stri
         if (eperson == null) {
             return null;
         }
-        return converter.fromModel(eperson);
+        return dsoConverter.fromModel(eperson);
     }
 
     @Override
     @PreAuthorize("hasAuthority('ADMIN')")
```

Comment:
I know this isn't part of your PR's changes, but it's a tad odd that EPerson patch requires *ADMIN* while all other DSO patch methods just require *WRITE* access.  I'm wondering (out loud) if we should change this?

Reply:
I agree, it is a bit odd. It seems at a minimum, 'WRITE' on the resource should be required for issuing any kind of patch. But I can see cases where certain types of edits should be subject to higher levels of access (e.g. certain EPerson properties). There is some good discussion about this in this PR: https://github.com/DSpace/DSpace/pull/2257

Reply:
Good point. We'll leave this for #2257 to decide, as that's the more appropriate place for it. I'll resolve this comment.